docs/guide/writing-tasks.md

```diff
@@ -895,9 +895,8 @@ Cirrus CI supports the `only_if` and `skip` keywords in order to provide such fl
 !!! tip "Skip CI Completely"
     Just include `[skip ci]` or `[skip cirrus]` in the first line or last line of your commit message in order to skip CI execution for a commit completely.
 
-    If you push multiple commits at the same time, only the first line of the last commit message will be checked for `[skip ci]`
-    or `[ci skip]`.
-    
+    If you push multiple commits at the same time, only the last commit message will be checked for `[skip ci]` or `[ci skip]`.
```

Comment:
Now it's missing that `first and last lines` of the commit message are checked for skipping.

Reply:
Isn't that already mentioned in the:

>Just include `[skip ci]` or `[skip cirrus]` in the first line or last line of your commit message in order to skip CI execution for a commit completely.

...on the line above?